CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+### Changed
+
+## [0.1.1]
```

Comment:
- Per [Semantic Versioning](https://semver.org/spec/v2.0.0.html) this seems like a minor version increment (adds functionality in a backwards compatible manner).
- The version number in [dbtenv/\_\_init\_\_.py](https://github.com/brooklyn-data/dbtenv/blob/main/dbtenv/__init__.py) needs to be updated.
- Let's put a date on this, and we can bump it if we miss that date.
```suggestion
## [1.1.0](https://github.com/brooklyn-data/dbtenv/compare/v1.0.0...v1.1.0) - 2021-07-14
```